catalogue/webapp/components/Download/DownloadBeta.js

```diff
@@ -0,0 +1,245 @@
+  }
+  .icon__shape {
+    fill: currentColor;
+  }
+`;
+const DownloadOptions = styled.div`
+  &.enhanced-styles {
+    border: ${props => `1px solid ${props.theme.colors.marble}`};
+    background: white;
+    box-shadow: 0 1px 6px 0 rgba(0, 0, 0, 0.3);
+    z-index: -1;
+    margin-top: ${props => `-${props.theme.spacingUnit}px`};
+    padding: ${props => `${props.theme.spacingUnit * 3}px`};
+    height: none;
+    opacity: 0;
+    transition: opacity 400ms;
+    position: absolute;
+  }
+  &.show {
+    z-index: 1;
+    opacity: 1;
+  }
+  li + li {
+    margin-top: ${props => `${props.theme.spacingUnit * 2}px`};
+  }
+  a {
+    color: ${props => props.theme.colors.green};
+    text-decoration: none;
+  }
+  .icon__canvas {
+    height: 1.3em;
+  }
+  .icon__shape {
+    fill: currentColor;
+  }
+`;
+
+type Work = Object;
+type Props = {|
+  work: Work,
+  iiifImageLocationUrl: string,
+  licenseInfo: ?LicenseData,
+  iiifImageLocationCredit: ?string,
+  iiifImageLocationLicenseId: ?LicenseType,
+|};
+
+const DownloadBeta = ({
+  work,
+  iiifImageLocationUrl,
+  licenseInfo,
+  iiifImageLocationCredit,
+  iiifImageLocationLicenseId,
+}: Props) => {
+  const [showDownloads, setShowDownloads] = useState(true);
+  const [useJavascriptControl, setUseJavascriptControl] = useState(false);
+  useEffect(() => {
+    setUseJavascriptControl(true);
+    setShowDownloads(false);
+  }, []);
+  return (
+    <div>
+      <div
+        className={classNames({
+          [font({ s: 'HNL5', m: 'HNL4' })]: true,
+        })}
+      >
+        {useJavascriptControl ? (
+          <h2 className="inline">
+            <DownloadButton
+              className={classNames({
+                [font({ s: 'HNM4' })]: true,
+                'flex-inline': true,
+                'flex--v-center': true,
+              })}
+              aria-controls="downloadOptions"
+              aria-expanded={showDownloads}
+              rotateIcon={showDownloads}
+              onClick={() => {
+                setShowDownloads(!showDownloads);
+              }}
+            >
+              <span className="flex-inline flex--v-center">
+                <span
+                  className={classNames({
+                    [spacing({ s: 1 }, { margin: ['right'] })]: true,
+                  })}
+                >
+                  Download this image
+                </span>
+                <Icon name="chevron" />
+              </span>
+            </DownloadButton>
+          </h2>
+        ) : (
+          <h2
+            className={classNames({
+              [font({ s: 'WB6', m: 'WB5' })]: true,
+              'work-details-heading': true,
+            })}
+          >
+            Download this image
+          </h2>
+        )}
+        <DownloadOptions
+          id="downloadOptions"
+          className={classNames({
+            [font({ s: 'HNM5', m: 'HNM4' })]: true,
+            'enhanced-styles': useJavascriptControl,
```

Comment:
Again, with the patterns, are we happy with this, if so, let's try to make it stick.